internal/cluster/cluster.go

```diff
@@ -863,8 +871,8 @@ func (m *Manager) DeleteClusterFiles(ctx context.Context, c *common.Cluster, obj
 			errors.Errorf(msg))
 	}
 	for _, fileName := range filesList {
```

Comment:
Can you deduplicate this logic? It feels like the only real difference between `DeleteClusterFiles` and `deleteClusterFiles` is the prefix we use.